### PR TITLE
Wm bookmarks

### PR DESCRIPTION
**Pull request by Will-D-M**
- Added bookmarks to user dashboard
- I was having trouble splitting out the methods like I did for followers (with individual models and calls in the facade), so I just put the helper method in the user model. Wanted to get it working and move on for now. 
- Added the additional setup code for travis CI, as recommended by Brian
- Coverage at 81.19%. Much much better!

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -37,6 +37,12 @@
 
   <section>
     <h1>Bookmarked Segments</h1>
+    <% current_user.bookmarks.each do |_tutorial, video| %>
+      <h3><%= video[0].tutorial.title %></h3>
```
Comment (kathleen-crowe) [inline app/views/users/show.html.erb:41]: should this be hardcoded to video[0] always?

Comment (Will-D-M) [reply app/views/users/show.html.erb:41]: Good question. Yes. "video" in each iteration is technically an array that consists of one video, so in order to turn that into an object, I am just calling the first index position. Could have also called .first or .last. Doesn't matter, but I had to convert it to the object somehow.

Comment (Will-D-M) [reply app/views/users/show.html.erb:41]: And this could most definitely be cleaner, but I just wanted to get it working for the time being.

Comment (kathleen-crowe) [reply app/views/users/show.html.erb:41]: Awesome!

**Pull request merged**